docker-compose.prod.yml

```diff
@@ -4,7 +4,7 @@ services:
 
   backend:
     volumes:
-      - './backend:/match4everyone-backend'
+      - backend-run:/backend/run
```

Comment:
We should probably create a subdirectory for logs and only bind that to a volume
and exclude the statics.